### ISSUE TITLE
Extra Screenshots showing how it would look with non GUI

### ISSUE THREAD

**Issue by zanona** [altercation/vim-colors-solarized#7]
I could notice there are some screenshots showing how the colorscheme should look but apparently they are all based on GUI (MacVim in this case) I've tried to run the colorscheme through VIM in terminal (terminal also using solarized color theme) but it looks slightly different then on MacVim ( which is exactly the same as the images shown in my case :)

So I would like to recommend uploading some images also showing how it would look like in pure VIM with no graphical interface.

![solarized on vim-noGUI](http://f.cl.ly/items/0S2O1Y0a2T0d2C1R0C06/Screen%20shot%202011-04-04%20at%2013.00.56.png)

This is how it looks for me, I don't know why but the line number are not faded as they should be, so in this case I don't know if I am doing something wrong or this is how it should look.

So the images could be a guide for us to know if we're achieving the right thing.

Also please bear with me since I was a TextMate, and because of this nice approach of your colorscheme It kind opened my eye and I decided to give a change and start to use VIM instead, so if I said something wrong, please my apologies. It's just I am sure you will get lots of feedback from users from different editors migrating because your great job there :)

**Comment by altercation** [comment]
This was due to Terminal.app handling only 8 colors and issues with how I ordered the syntax highlighting arguments. I've spent a lot of today working on specifically this issue and, on the current Terminal.app themes available from the main [Solarized repository](https://github.com/altercation/solarized) (I recommend downloading those again as I've created a new ansi color only Terminal.app theme).

It is working great here with the basic ansi Solarized terminal.app theme and the very recent solarized.vim. I'd clone or pull the current repo and give the new code and themes a shot.

Please do get back to me and let me know if this fixes the issue!

**Comment by zanona** [comment]
Wow, this is really great

I got it working perfectly now.

I had indeed to set `let g:solarized_termcolors=16` to get it working properly before but even so it had some discrepancies from the original design. Fortunately now it works just fine in 256 :) thanks a lot.

The only think I had to do though, is set `let g:solarized_bold=0` because the type didn't look nice and it seems and as I don't get bold fonts on macvim, it looked better :) 

thanks again man!
